resources/core/values.yaml

```diff
@@ -21,7 +21,7 @@ global:
     # - Change the value below to the branch from your fork if you want to point to documentation sources from your fork
     # - Change the value below to the release branch during the release
     # Example: clusterAssetGroupsVersion: "release-0.9"
-    clusterAssetGroupsVersion: release-1.14
+    clusterAssetGroupsVersion: release-1.14.0-rc2
```

Comment:
This should stay same as the branch name `release-1.14`.

Reply:
resolved